frontend/components/form/InterfaceCompliance.tsx

```diff
@@ -60,7 +60,23 @@ const InterfaceComplianceForm = ({
         </p>
       </div>
       <div className="interface-bb-selector">
-        <p className="text-18">{format('form.select_building_blocks.label')}</p>
+        <p className="text-18">{format('form.select_building_blocks.top.label',
+          {
+            Gitbook: <a className="software-attributes-title-edit-link no-underline"
+              href={'https://govstack.gitbook.io/specification'}>Gitbook</a>
```

Comment:
We use "Gitbook" hardcoded in few places. For the future I would consider treating it as a standard translation. If something changes it will be easier to change it in all places at the same time.

I also think it should be "GitBook" instead (capital B), that's the official name.